src/core/sentences/global-sentences.storage.ts

```diff
@@ -0,0 +1,29 @@
+import { SentenceEntityGlobal } from './global-sentences.model';
+
+export const defaultSentences: SentenceEntityGlobal[] = [
+  {
+    verb: 'win',
+    sentence: `Brazil _______ five world cups, it’s incredible!`,
+    rigthAnswer: 'won',
```

Comment:
righAnswer should be:

Present, Past or Participle

You can create a type

```
type Tenses = 'Present' | 'Past' | 'Participle'
```

Why it cannot be the verb itself? e.g. Win Won Won in this case is the past, but what if the user chooses the past participle, even if it's the same text it's wrong

Reply:
done